### PR TITLE
Restructured Code

### PR DESCRIPTION
**Pull request by eclecticexistential**
Added while loops to fill arrays with ids to use throughout the program. Created functions that handle toggling upgrades.

### CODE REVIEW

**Comment by StevenCopeland** [comment]
Hello, @eclecticexistential! This is your first Pull Request that will be reviewed by [Ebert](https://ebertapp.io/), an automatic Code Review service. It will leave comments on this diff with potential issues and style violations found in the code as you push new commits. You can also see all the issues found on this Pull Request on its [review page](https://ebertapp.io/github/StevenCopeland/FILEPLAN/pulls/36). Please check our [documentation](https://docs.ebertapp.io) for more information.

File: script.js [hunk 1]
```diff
@@ -1,115 +1,79 @@
...
+	hoverThem(byteUpgradeSecArray[2],upgradeArray[4]);
+	hoverThem(byteUpgradeSecArray[3],upgradeArray[5]);
+	hoverThem(byteUpgradeSecArray[4],upgradeArray[6]);
+	hoverThem(byteUpgradeSecArray[5],upgradeArray[7]);
+	hoverThem(byteUpgradeIdeaArray[0],upgradeArray[8]);
+	hoverThem(byteUpgradeIdeaArray[1],upgradeArray[9]);
+	hoverThem(byteUpgradeIdeaArray[2],upgradeArray[10]);
+	hoverThem(byteUpgradeIdeaArray[3],upgradeArray[11]);
+	hoverThem(byteUpgradeIdeaArray[4],upgradeArray[12]);
+	hoverThem(byteUpgradeIdeaArray[5],upgradeArray[13]);
```
Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'upgradeArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'byteUpgradeIdeaArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 2]
```diff
@@ -1,115 +1,79 @@
...
+	hoverThem(byteUpgradeSecArray[1],upgradeArray[3]);
+	hoverThem(byteUpgradeSecArray[2],upgradeArray[4]);
+	hoverThem(byteUpgradeSecArray[3],upgradeArray[5]);
+	hoverThem(byteUpgradeSecArray[4],upgradeArray[6]);
+	hoverThem(byteUpgradeSecArray[5],upgradeArray[7]);
+	hoverThem(byteUpgradeIdeaArray[0],upgradeArray[8]);
+	hoverThem(byteUpgradeIdeaArray[1],upgradeArray[9]);
+	hoverThem(byteUpgradeIdeaArray[2],upgradeArray[10]);
+	hoverThem(byteUpgradeIdeaArray[3],upgradeArray[11]);
+	hoverThem(byteUpgradeIdeaArray[4],upgradeArray[12]);
```
Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'upgradeArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'byteUpgradeIdeaArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 3]
```diff
@@ -1,115 +1,79 @@
...
+	hoverThem(byteUpgradeSecArray[0],upgradeArray[2]);
+	hoverThem(byteUpgradeSecArray[1],upgradeArray[3]);
+	hoverThem(byteUpgradeSecArray[2],upgradeArray[4]);
+	hoverThem(byteUpgradeSecArray[3],upgradeArray[5]);
+	hoverThem(byteUpgradeSecArray[4],upgradeArray[6]);
+	hoverThem(byteUpgradeSecArray[5],upgradeArray[7]);
+	hoverThem(byteUpgradeIdeaArray[0],upgradeArray[8]);
+	hoverThem(byteUpgradeIdeaArray[1],upgradeArray[9]);
+	hoverThem(byteUpgradeIdeaArray[2],upgradeArray[10]);
+	hoverThem(byteUpgradeIdeaArray[3],upgradeArray[11]);
```
Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'upgradeArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'byteUpgradeIdeaArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 4]
```diff
@@ -1,115 +1,79 @@
...
+	hoverThem($("#byteUpgradeSec2"),upgradeArray[1]);
+	hoverThem(byteUpgradeSecArray[0],upgradeArray[2]);
+	hoverThem(byteUpgradeSecArray[1],upgradeArray[3]);
+	hoverThem(byteUpgradeSecArray[2],upgradeArray[4]);
+	hoverThem(byteUpgradeSecArray[3],upgradeArray[5]);
+	hoverThem(byteUpgradeSecArray[4],upgradeArray[6]);
+	hoverThem(byteUpgradeSecArray[5],upgradeArray[7]);
+	hoverThem(byteUpgradeIdeaArray[0],upgradeArray[8]);
+	hoverThem(byteUpgradeIdeaArray[1],upgradeArray[9]);
+	hoverThem(byteUpgradeIdeaArray[2],upgradeArray[10]);
```
Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'upgradeArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'byteUpgradeIdeaArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 5]
```diff
@@ -1,115 +1,79 @@
...
+	hoverThem($("#byteUpgradeSec1"),upgradeArray[0]);
+	hoverThem($("#byteUpgradeSec2"),upgradeArray[1]);
+	hoverThem(byteUpgradeSecArray[0],upgradeArray[2]);
+	hoverThem(byteUpgradeSecArray[1],upgradeArray[3]);
+	hoverThem(byteUpgradeSecArray[2],upgradeArray[4]);
+	hoverThem(byteUpgradeSecArray[3],upgradeArray[5]);
+	hoverThem(byteUpgradeSecArray[4],upgradeArray[6]);
+	hoverThem(byteUpgradeSecArray[5],upgradeArray[7]);
+	hoverThem(byteUpgradeIdeaArray[0],upgradeArray[8]);
+	hoverThem(byteUpgradeIdeaArray[1],upgradeArray[9]);
```
Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'upgradeArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'byteUpgradeIdeaArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 6]
```diff
@@ -1,115 +1,79 @@
...
+	
+	hoverThem($("#byteUpgradeSec1"),upgradeArray[0]);
+	hoverThem($("#byteUpgradeSec2"),upgradeArray[1]);
+	hoverThem(byteUpgradeSecArray[0],upgradeArray[2]);
+	hoverThem(byteUpgradeSecArray[1],upgradeArray[3]);
+	hoverThem(byteUpgradeSecArray[2],upgradeArray[4]);
+	hoverThem(byteUpgradeSecArray[3],upgradeArray[5]);
+	hoverThem(byteUpgradeSecArray[4],upgradeArray[6]);
+	hoverThem(byteUpgradeSecArray[5],upgradeArray[7]);
+	hoverThem(byteUpgradeIdeaArray[0],upgradeArray[8]);
```
Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'upgradeArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'byteUpgradeIdeaArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 7]
```diff
@@ -1,115 +1,79 @@
...
+	}
+	
+	hoverThem($("#byteUpgradeSec1"),upgradeArray[0]);
+	hoverThem($("#byteUpgradeSec2"),upgradeArray[1]);
+	hoverThem(byteUpgradeSecArray[0],upgradeArray[2]);
+	hoverThem(byteUpgradeSecArray[1],upgradeArray[3]);
+	hoverThem(byteUpgradeSecArray[2],upgradeArray[4]);
+	hoverThem(byteUpgradeSecArray[3],upgradeArray[5]);
+	hoverThem(byteUpgradeSecArray[4],upgradeArray[6]);
+	hoverThem(byteUpgradeSecArray[5],upgradeArray[7]);
```
Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'upgradeArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

Comment (StevenCopeland) [inline script.js]: Possible undefined variable: 'byteUpgradeSecArray' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)

**Comment by StevenCopeland** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 74 possible new issues (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/StevenCopeland/FILEPLAN/pulls/36.

**Comment by eclecticexistential** [comment]
Woah.

**Comment by StevenCopeland** [comment]
Sorry! You had to go through all of the reviews(some of which contradict each other)!

**Comment by StevenCopeland** [comment]
Merging to master!

**Pull request merged**